apps/meteor/ee/app/livechat-enterprise/server/api/priorities.ts

```diff
@@ -3,7 +3,7 @@ import { findPriorities, findPriorityById } from './lib/priorities';
 
 API.v1.addRoute(
 	'livechat/priorities',
-	{ authRequired: true },
+	{ authRequired: true, permissionsRequired: { GET: { permissions: ['manage-livechat-priorities', 'view-l-room'], operation: 'hasAny' } } },
```

Comment:
Since we have a permission check here, so can we remove it from `findPriorities` to avoid re-checking?